roles/common/tasks/main.yml

```diff
@@ -7,6 +7,11 @@
   include_vars: "{{ item }}"
   loop: "{{ lookup('first_found', os_fallback, errors='ignore', wantlist=True) }}"
 
+- name: Set Repo/Solr shared secret
+  set_fact:
+    reposearch_shared_secret: "{{ reposearch_shared_secret | default(lookup('password', '/dev/null')) }}"
```

Comment:
can we just make it failing if a secret is not provided when required?

Reply:
That was the other option but I decided to not follow it as we will add more and more credentials vars as we progress the "hardening" of the playbook. I'm afraid the pre-requisites to get a first play to run would be way too big.